main.cpp

```diff
@@ -270,6 +429,28 @@ Thing 6) Door
     2) Close 
     3) Get status (open or closed)
  */
+struct Door
+{
+
+    //Height in metres
+    double doorHeight = 2.4;
+    //Width in metres (float)
+    double doorWidth = 1.2;
+    //Speed of movement in metres per second
+    double speedOfDoorMovement = 0.5;
+    //Colour (std::string)
+    std::string doorColour = "grey";
+    //Whether door is closed (false) or open (true)
+    bool doorClosureStatus = false;
+
+    //Open
+    void openDoor();
+    //Close 
+    void closeDoor();
+    //Get status (open or closed)
+    bool doorStatus(bool doorClosureStatus); //returns true or false according to whether door closed/open
```

Comment:
We already know it is a door because of the class name. You do not need to include the word door in any of these variable or function names.
- [ ] revise